src/chart/Column.js

```diff
@@ -31,46 +31,62 @@
 
         this._palette = this._palette.switch(this.paletteID());
 
-        var column = this.svgData.selectAll(".columnRect")
+        var insideScale = d3.scale.ordinal();
+
+        var column = this.svgData.selectAll(".rectSubject")
```

Comment:
rename "rectSubject" to something more suitable (the data won't always be about subjects) - ".dataRow"?